ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/InMemoryKVParticipantStateIT.scala

```diff
@@ -319,4 +391,39 @@ class InMemoryKVParticipantStateIT extends AsyncWordSpec with AkkaBeforeAndAfter
     }
   }
 
+  "return update [0,1] with beginAfter=[0,0]" in {
+    val ps = new InMemoryKVParticipantState(participantId)
+    val rt = ps.getNewRecordTime()
+
+    val sourceDescription = Some("provided by test")
+    val archives = List(
```

Comment:
Minor: this fixture seems to resurface every now and then, you may want to pull it out of individual test, making it a `def` if needed.

Reply:
Done